cpp/zotaut/Makefile

```diff
@@ -0,0 +1,30 @@
+LIB := ../lib
+INC = $(LIB)/include
```

Comment:
Shouldn't this be the other way 'round?  I mean using "=" on the first line an ":=" on the second line?

Reply:
The makefile was copied from the pipelines subdir. The other makefile (e.g. cpp/Makefile) also start with `LIB := ../lib`

Reply:
I was afraid of that!  According to what I found on Stackoverflow line 2 seems suspicious.  After reading https://www.gnu.org/software/make/manual/html_node/Setting.html it seems that this works but is probably not optimal.